HospitalSystem.ConsoleUI/Secretary/CrudUI.cs

```diff
@@ -0,0 +1,311 @@
+using System.Text.RegularExpressions;
+using HospitalSystem.Core;
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+namespace HospitalSystem.ConsoleUI;
+[System.Serializable]
+
+public class CrudUI : ConsoleUI
+{
+
+    public CrudUI(Hospital hospital) : base(hospital)
+    {
+        // _loadedBatches = _hospital.EquipmentRepo.GetAll().ToList();
```

Comment:
Again with the Serializable UI. That should not be serializable!! Also commented out useless stuff